mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingsaccount/SavingsAccountFragment.java

```diff
@@ -119,7 +122,12 @@ public static SavingsAccountFragment newInstance(int clientId) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         ((MifosBaseActivity) getActivity()).getActivityComponent().inject(this);
-        if (getArguments() != null) clientId = getArguments().getInt(Constants.CLIENT_ID);
+        Bundle arguments = getArguments();
+        if (arguments != null) {
+            mIsGroupAccount = arguments.containsKey(Constants.GROUP_ID);
```

Comment:
Using the containsKey is like, putting our self on dark or If we can lot's of things are present in bundle and we just want to check it's present or not But here we know what we are going to get or not. So better use
```java
arguments.getBoolean(String key)
```